src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -202,6 +202,10 @@ private static (string repository, string branch) GetValidationServiceParameters
 
     private static string GetHostName(string siteName)
```

Comment:
It is theoretically possible that multiple docsets in a repo belong to different sites. Can we verify whether this is the case by taking a look at existing provisioning data?

Reply:
If it is true that we do have such cases, consider passing in the `siteName` -> `hostName` mapping instead of the `hostName` itself.

Reply:
Since this is only for PPE

Consider rename to DOCFX_PPE_DEFAULT_DOMAIN_HOST_NAME

Reply:
TODO:

add the env var from upstream